Jenkinsfile.content

```diff
@@ -1,6 +1,23 @@
+@Library('va.gov-devops-jenkins-lib') _
+
 node('vetsgov-general-purpose') {
+  refStatusState = ""
+  ref = params.ref
+  if (ref == "") {
+    ref = getLatestGitRef("vets-website", branch: "master")
+  }
+
+  refStatusState = getGithubRefStatusState("vets-website", ref)
+  if ("${refStatusState}" != "SUCCESS") {
```

Comment:
We might want to tweak the message text if the status is in progress (or whatever the yellow dot icon means).

Reply:
Yeah, how about `Web content refresh on ${params.env} aborted due to '${refStatusState}' status on vets-website commit ${ref}.` or do you think it should be more custom per status?

Reply:
That sounds good to me!